tower_cli/cli/transfer/receive.py

```diff
@@ -104,6 +105,12 @@ def export_assets(self, all, asset_input):
                         exported_asset[common.ASSET_RELATION_KEY][relation] =\
                             common.extract_schedules(asset)['items']
 
+                # If this asset type is in the RESOURCE_FIELDS of the Role object than export its roles
+                if asset_type in RESOURCE_FIELDS:
+                    if common.ASSET_RELATION_KEY not in exported_asset:
+                        exported_asset[common.ASSET_RELATION_KEY] = {}
+                    exported_asset[common.ASSET_RELATION_KEY]['roles'] = common.extract_roles(asset)['items']
```

Comment:
will this copy system roles?

Reply:
Per our conversation, system roles won't be copied here. They will be granted at the user level.